.github/scripts/branded_preview.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -eou pipefail
+
+foascli versions -s v2.json --env "${target_env:?}" -o versions.json
+# Load versions from versions.json
+versions=()
+
+# Read versions from versions.json into an array
+while IFS= read -r version; do
+    versions+=("$version")
+done < <(jq -r '.[]' versions.json)
+
+all_urls=(
+   "https://raw.githubusercontent.com/mongodb/openapi/${target_env:?}/openapi/v2.json"
```

Comment:
do we need to have a target_branch that is equal to target_env except for `prod`, where it should be `main`?

Reply:
good catch! yes, indeed. I will update it

Reply:
updated